<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewConnectionDlg</class>
 <widget class="QDialog" name="NewConnectionDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>349</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>294</width>
    <height>349</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>294</width>
    <height>349</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create New Connection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 199, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../../../Resources/Images/resources.qrc">:/Images/edit.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>MongoDB Connection</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="settingsTab">
        <attribute name="title">
         <string>Settings</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Enter Basic Settings:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Name:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditConnectionName</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditConnectionName"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Server:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditServer</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditServer"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Port:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditPort</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditPort">
             <property name="maximumSize">
              <size>
               <width>113</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>27017</string>
             </property>
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Username:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditUsername</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditUsername"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Password:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditPassword</cstring>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEditPassword">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Database(s):</string>
             </property>
             <property name="buddy">
              <cstring>lineEditDatabases</cstring>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEditDatabases"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="sshTab">
        <attribute name="title">
         <string>SSH Tunnel</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBoxUseSSHTunnel">
             <property name="text">
              <string>Use SSH Tunnel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>SSH Host:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditSSHHost</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditSSHHost">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Username:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditSSHUsername</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditSSHUsername">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Port:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditSSHPort</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditSSHPort">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>113</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>22</string>
             </property>
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="lineEditSSHPassphrase">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Passphrase:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditSSHPassphrase</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="comboBoxSSHSecurity">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>111</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>PublicKey</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Security:</string>
             </property>
             <property name="buddy">
              <cstring>comboBoxSSHSecurity</cstring>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Private Key:</string>
             </property>
             <property name="buddy">
              <cstring>lineEditSSHPrivateKey</cstring>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEditSSHPrivateKey">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonOpenKeyFile">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="maximumSize">
                <size>
                 <width>31</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(199, 199, 255);</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="pushButtonTestConnection">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Test</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSave">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../Resources/Images/resources.qrc"/>
  <include location="../../../../Resources/Images/resources.qrc"/>
  <include location="../../../../Resources/Images/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>NewConnectionDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseSSHTunnel</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxSSHSecurity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseSSHTunnel</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditSSHHost</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseSSHTunnel</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditSSHUsername</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseSSHTunnel</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditSSHPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseSSHTunnel</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditSSHPassphrase</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseSSHTunnel</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditSSHPrivateKey</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxUseSSHTunnel</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButtonOpenKeyFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
